src/components/LeaderboardRow.tsx

```diff
@@ -122,13 +129,11 @@ export const WinLoseStreak = ({
     return <span class="pr-2 text-2xl">💤</span>;
   }
   if (streak && streak === 5) {
-    return <span class="pr-2 text-2xl">{isWinStreak ? "🤑" : "🗑️"}</span>;
+    return <span class="pr-2 text-2xl">{isWinStreak ? "🤯" : "🗑️"}</span>;
```

Comment:
mangler at reverte ikon tilbage

```suggestion
    return <span class="pr-2 text-2xl">{isWinStreak ? "🤑" : "🗑️"}</span>;
```